velox/functions/sparksql/Register.cpp

```diff
@@ -175,6 +176,12 @@ void registerFunctions(const std::string& prefix) {
 
   registerRegexpReplace(prefix);
 
+  registerJsonType();
+  registerFunction<JsonObjectKeysFunction, Array<Varchar>, Json>(
+      {prefix + "json_object_keys"});
+  registerFunction<JsonObjectKeysFunction, Array<Varchar>, Varchar>(
```

Comment:
Spark does not have a json type. I wonder if only varchar input type is needed for Spark. Could you confirm?
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala#L1036

Reply:
Yes, spark do not have json type, I will remove it